ui/js/models/resultsets_store.js

```diff
@@ -1023,7 +1023,7 @@ treeherder.factory('ThResultSetStore', [
                             });
                         });
                         platform.groups = _.sortBy(platform.groups, function(group){
-                            return jobGroupOrder[group.symbol];
+                            return jobGroupOrder[group.symbol.split("[",1)];
```

Comment:
Tapesh, this works great, but let's add a comment telling why we're doing this.  This will also make it easier to find occurrences if we decide to change using the ``[`` in the future.  Perhaps wording like this.

    // remove any visual additions for Tier when sorting.